source/includes/_dataset_status.md

```diff
@@ -0,0 +1,307 @@
+# Dataset status
+
+## The dataset status object
+
+A dataset status describe the current state of a dataset, if it's published or not and the running operation. It is a finite state machine, with the following properties:
+
+* It is in a single state at a time
+* The list of possible states is fully known
+* The list of all transitions is fully known, each with the actions and conditions that can trigger them
+
+Here is the full state machine description. 
+
+![Dataset states machine](dataset_status_states.png "Dataset states machine")
```

Comment:
I'm surprised you can't go directly from deleting to idle without going through processing. Also, the transitions at the right end of the schema are mixed up, it's a little bit confusing.